docs/introduction/Sidechain-SDK-extension.rst

```diff
@@ -127,13 +127,13 @@ That base class provides the following data by default:
 
   proposition of type P long value
 
-If the box type is a Coin-Box then this value is required and will contain data such as coin value. In the case of a Non-Coin box it will be used in custom logic only. As a common practice for non-Coin box you can set it always equal to 1 
+If the box type is a Coin-Box, then this value is required and will contain data such as coin value. In the case of a Non-Coin box it will be used in custom logic only. As a common practice for non-Coin box you can set it always equal to 1 
```

Comment:
@zennermac - LGTM

Reply:
@zennermac - If the box type is a CoinBox, then this value is required and will contain data such as coin value. In the case of a Non-CoinBox it will be used in custom logic only. As a common practice for Non-CoinBox you can set it always equal to 1